src/library/scala/math/package.scala

```diff
@@ -215,9 +218,37 @@ package object math {
     */
   def signum(x: Long): Long     = java.lang.Long.signum(x)
   /** @group signum */
-  def signum(x: Float): Float   = java.lang.Math.signum(x)
+  def signum(f: Float): Float   = java.lang.Math.signum(f)
```

Comment:
The parameter name cannot be changed with a deprecation annotation. This is out of scope for this PR don't you think?